pkg/ingester/lifecycle_test.go

```diff
@@ -31,22 +31,17 @@ const (
 
 func defaultIngesterTestConfig() Config {
 	consul := ring.NewInMemoryKVClient()
-	return Config{
-		LifecyclerConfig: ring.LifecyclerConfig{
-			KVClient: consul,
-
-			NumTokens:       1,
-			HeartbeatPeriod: 5 * time.Second,
-			ListenPort:      func(i int) *int { return &i }(0),
-
-			Addr: "localhost",
-			ID:   "localhost",
-		},
-
-		FlushCheckPeriod:  99999 * time.Hour,
-		MaxChunkIdle:      99999 * time.Hour,
-		ConcurrentFlushes: 1,
-	}
+	cfg := Config{}
```

Comment:
I'm not particularly against the new format, but it's eyestrain to compare before and after, so I'm wondering why it had to change?

Reply:
The defaults for `MaxSeriesPerQuery` and `MaxSamplesPerQuery` needed to be initialised; instead of adding another place where it is initialised, I wanted to use the new `util.DefaultValues` function as it will be less brittle in the future, but it prevents the old style.

Reply:
ah, it was on the next line, but I hadn't spotted it